apps/web/src/hooks/use-novu.ts

```diff
@@ -1,5 +1,5 @@
 import { useContext, useEffect } from 'react';
-import { APP_ID, ENV, WIDGET_EMEBED_PATH } from '../config';
+import { APP_ID, ENV, WIDGET_EMBED_PATH } from '../config';
```

Comment:
Rollback this one please. This can affect current users settings and break their systems.